From af7992788429791ef77ffee6f5278ba1e6a837b5 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Thu, 8 Jun 2023 11:49:26 +0200
Subject: [PATCH 0340/1141] arm64: dts: st: enable cryp1 on stm32mp257f-ev1
 board

Set STM32 CRYP peripheral enabled.

Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Change-Id: Id567ca7443c87b6b96fc1b83578719cee464236a
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/311580
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 4 ++++
 1 file changed, 4 insertions(+)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -130,6 +130,10 @@
 	status = "okay";
 };
 
+&cryp1 {
+	status = "okay";
+};
+
 &csi2host {
 	status = "okay";
 	ports {
